src/harness/sas.py

```diff
@@ -76,6 +75,40 @@ def _Request(url, request, ssl_cert, ssl_key):
   logging.debug('Response:\n' + body)
   return json.loads(body)
 
+def _RequestGet(url, ssl_cert, ssl_key):
```

Comment:
Yes, all the Injection methods are HTTP POST. But SAS - SAS Testcases like ARE.1, ARE.3, SIR.1, SIR.3 and so on requires HTTP GET Request.